python/pip_install/private/srcs.bzl

```diff
@@ -0,0 +1,19 @@
+"""A generate file containing all source files used for `@rules_python//python/pip_install:pip_repository.bzl` rules"""
```

Comment:
It would be good to mention how to regenerate this file, and when that is needed.